packages/@ama-sdk/schematics/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@ama-sdk/schematics",
+  "version": "0.0.0-placeholder",
+  "description": "Swagger specification SDK Generator by schematics",
+  "scripts": {
+    "nx": "nx",
+    "ng": "yarn nx",
+    "test": "yarn nx test ama-sdk-schematics",
+    "prepare:build:builders": "yarn cpy 'schematics/**/*.json' 'schematics/**/templates/**' 'schematics/**/*.jar' dist/schematics && yarn cpy '{package,collection}.json' dist",
+    "build": "yarn nx build ama-sdk-schematics",
+    "build:builders": "tsc -b tsconfig.builders.json --pretty",
+    "install-swagger-cli": "mvn install:install-file -DgroupId=io.swagger -DartifactId=swagger-codegen-cli -Dversion=2.4.0-AMADEUS -Dpackaging=jar -Dfile=./schematics/resources/swagger-codegen-cli.jar",
+    "build:swagger": "yarn install-swagger-cli && run-p 'build:swagger-*'",
+    "build:swagger-typescript-gen": "mvn clean package -f ./schematics/typescript/core/swagger-codegen-typescript/pom.xml",
+    "build:swagger-java-client-gen": "mvn clean package -f ./schematics/java/client-core/swagger-codegen-java-client/pom.xml",
+    "prepare:publish": "prepare-publish ./dist"
+  },
+  "schematics": "./collection.json",
+  "keywords": [
+    "schematics",
+    "sdk"
+  ],
+  "peerDependenciesMeta": {
+    "@angular/compiler": {
+      "optional": true
+    }
+  },
+  "peerDependencies": {
+    "@angular-devkit/core": "~15.2.0",
+    "@angular-devkit/schematics": "~15.2.0",
+    "@schematics/angular": "~15.2.0"
+  },
+  "dependencies": {
+    "@angular-devkit/core": "~15.2.0",
+    "@angular-devkit/schematics": "~15.2.0",
+    "@o3r/dev-tools": "workspace:^",
+    "minimatch": "^6.1.6",
+    "rxjs": "^7.4.0",
+    "sway": "^2.0.6",
+    "tslib": "^2.4.1"
+  },
+  "devDependencies": {
+    "@ama-sdk/core": "workspace:^",
```

Comment:
missing in optional `peer` as well. Is it is not needed to the `ng-add` but to make sure the regen is aligned on the current core version